frontend/pages/admin/UserManagementPage/_styles.scss

```diff
@@ -78,28 +77,15 @@
           width: auto;
         }
         @media (min-width: $break-990) {
-          .role__header {
-            border-right: 1px solid $ui-fleet-blue-15;
-          }
           .teams__header {
             display: table-cell;
             width: $col-md;
-            border-right: none;
           }
         }
         @media (min-width: $break-1400) {
-          .teams__header {
-            border-right: 1px solid $ui-fleet-blue-15;
-          }
           .status__header {
             display: table-cell;
             width: $col-md;
-            border-right: none;
-          }
-        }
-        @media (min-width: $break-1600) {
-          .teams__header {
-            border-right: 1px solid $ui-fleet-blue-15;
```

Comment:
Lots of random old styling for the tables. Will be good to go through and clean this up.